src/Microsoft.Identity.Web.GraphServiceClient/Microsoft.Identity.Web.GraphServiceClient.csproj

```diff
@@ -12,9 +12,12 @@
     <PackageValidationBaselineVersion>2.12.2</PackageValidationBaselineVersion>
   </PropertyGroup>
 
-  <ItemGroup>
-    <PackageReference Include="Microsoft.Graph" Version="5.34.0" />
-    <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="8.0.0" />
```

Comment:
When separating this out by target, I couldn't find a reason to include DiagnosticSource, it is already a transitive dep of Microsoft.Kiota.Authentication.Azure